### DISCUSSION TITLE
GtiHub mobile app markdown parsing error

### DISCUSSION THREAD

**Discussion by do-me** [reinterpretcat/vrp#124] [General]
I noticed that you sometimes use a different markdown syntax in the release notes so that it becomes less readable. 

![image](https://github.com/reinterpretcat/vrp/assets/47481567/c7b32c36-5483-42d7-bae7-84a17e901734)

On mobile it becomes a problem as for some reason with the current style nothing is displayed: 
![image](https://github.com/reinterpretcat/vrp/assets/47481567/9abec167-e518-4542-af4e-37cb382768cb)

vs. Desktop
![image](https://github.com/reinterpretcat/vrp/assets/47481567/e3f6e772-9f7e-4f23-b34f-a0b569d403fb)

**Comment by reinterpretcat** [comment]
hm, interesting. I was not aware about problems on mobile. The message is just copied automatically from commit message. Thanks for reporting!

**Reply by do-me** [reply]
By the way, the error still persists on mobile apps (Android/iOS). If you use maybe "-" instead of "*" it might work? Shouldn't alter the way it's displayed.

**Reply by reinterpretcat** [reply]
yeah, maybe will try next time: completely forgot about this aspect..
